index.html

```diff
@@ -1113,6 +1113,58 @@ <h2>Fragment</h2>
 
       </section>
 
+      <section>
+        <h2>Relative DID URLs</h2>
+
+        <p>
+A relative DID URL is any URL value in a <a>DID document</a> that does not start
+with <code>did:&lt;method-name>:&lt;method-specific-id></code>. More
+specifically, it is any URL value that does not start with the ABNF defined  in
+Section <a href="#generic-did-syntax"></a>. The contents of the URL typically
+refers to a resource in the same <a>DID document</a>. Relative DID URLs MAY
+contain relative path components, query parameters, and fragment identifiers.
+        </p>
+
+        <p>
+When resolving a relative DID URL reference, the algorithm specified in <a
+data-cite="RFC3986#section-5">RFC3986 Section 5: Reference Resolution</a>  MUST
+be used. The <strong>base URI</strong> value is the <a>DID subject</a>.  The
```

Comment:
```suggestion
be used. The <strong>base URI</strong> value is the <a>DID value</a>.  The
```

Reply:
I agree with @dhh1128

Reply:
Hmm, DID value isn't defined in our terminology. We had this discussion a while ago to not define "XYZ value" because where do we stop if we start there? Just about every property in the spec has a corresponding value. I've reworded this to address @dhh1128 core concern c8a2508970c4c1a435f72d3b0b9dfb5a136c33ed.